src/customize-pop/dock-settings.md

```diff
@@ -0,0 +1,103 @@
+# Dock Settings
+
+This section describes settings located in the `Dock` tab for modifying Dock visibility behavior and Dock icons.
+
+![Customize Dock](/images/customize-desktop/customize-dock.png)
+
+## Enable Dock
+
+Disable or enable the Dock's visibility.
+
+**Dock Enabled**
+
+![Dock Enabled](/images/customize-desktop/dock-enabled.png)
+
+**Dock Disabled**
+
+![Dock Disabled](/images/customize-desktop/dock-disabled.png)
+
+## Dock Options
+
+Dock options include settings to enable icons for utilities (such as the Launcher, Applications menu, and viewing workspaces), configure focusing behavior when an application's Dock icon is clicked, and enabling visual characteristics of the Dock.
+
+![Dock Options](/images/customize-desktop/dock-options.png)
+
+### Extend Dock to Edges of the Screen
+
+Lengthen the Dock's edges so that it appears as a panel that extends to the screen edges.
+
+![Extend Dock Edges](/images/customize-desktop/extend-dock-edges.png)
+
+### Show Launcher Icon in Dock
+
+Display an icon that starts the Launcher when clicked.
+
+![Launcher Icon](/images/customize-desktop/launcher-icon.png)
+
+### Show Workspaces Icon in Dock
+
+Display an icon that activates the workspaces and windows view when clicked.
+
+![Workspaces Icon](/images/customize-desktop/workspaces-icon.png)
+
+### Show Applications Icon in Dock
+
+Display an icon that launches the Application menu when clicked.
```

Comment:
The menu is called `Applications`, not `Application`. If you leave off the `s`, you're no longer using its proper name and it should not be capitalized.

Reply:
I meant to add the "s"